src/meshes/mesh_io.jl

```diff
@@ -222,7 +222,7 @@ end
 
 # TODO: Implement this function as soon as there is support for this in `t8code`.
 function save_mesh_file(mesh::T8codeMesh, output_directory, timestep, mpi_parallel)
-    @warn "Mesh file output not supported yet for `T8codeMesh`."
+    @error "Mesh file output not supported yet for `T8codeMesh`."
```

Comment:
Are you aware that this doesn't throw an exception and just continues with the remaining function body?

Reply:
Didn't know. Changed it to the error function. This should throw an exception with a strack trace.